WalletWasabi/WabiSabi/Models/DecompositionAlgs/Coin.cs

```diff
@@ -0,0 +1,40 @@
+using NBitcoin;
+using System;
+
+namespace WalletWasabi.WabiSabi.Models.DecompositionAlgs
+{
+	public record Coin : IComparable
+	{
+		public Money Amount { get; init; } = Money.Zero;
+		public const int OutputVbytes = 31;
+		public const decimal InputVbytes = 68.5m;
+
+		private int HashCode { get; } = new Guid().GetHashCode();
```

Comment:
I added it because of In[40]

```
class Coin(int):
    def __hash__(self):
        """
        coins have a unique identify even if their face value is the same.
        this is useful for constructing sets of coins.
        """
        return id(self)

```

Reply:
This was entirely removed as there is no Coin class.